parameters/parser/ParamsParser.go

```diff
@@ -289,6 +313,23 @@ func (g *ParsedGlobalParams) GetVersionedGlobalParamsByHeight(btcHeight uint64)
 	return nil
 }
 
+// FindLastStakingCap finds the last staking cap that is not zero
+// it returns zero if not non-zero value is found
+func FindLastStakingCap(prevVersions []*VersionedGlobalParams) uint64 {
```

Comment:
nit: this probably does not need to be public method ?

Reply:
It is also used in tests